server/src/main/scala/actors/daily/TerminalDayFlightActor.scala

```diff
@@ -78,11 +81,33 @@ class TerminalDayFlightActor(
     restorer.finish()
   }
 
+  def getRedListCount(redListPassengers: RedListPassengers, now: () => SDateLike): Set[String] = {
+    val neboArrivalActor: ActorRef = context.system.actorOf(NeboArrivalActor.props(redListPassengers, now))
+    val neboArrivals: NeboArrivals = Await.result(neboArrivalActor.ask(GetState)(Timeout(60 seconds)).mapTo[NeboArrivals], 5 seconds)
```

Comment:
Let's improve this by removing the `Await.result`, as it's strongly discouraged to block threads - see https://docs.scala-lang.org/overviews/core/futures.html

Reply:
In fact, we can get rid of this code altogether if we streamline the import process - details to follow